caim_base/models/animals.py

```diff
@@ -16,8 +16,8 @@
 
 
 class AnimalType(models.TextChoices):
-    DOG = "DOG", "Dog"
-    CAT = "CAT", "Cat"
+    DOG = "DOG", "dog"
```

Comment:
I dont know if this got mixed up in the other PR, but as I wrote there, I would leave it capitalised here for select and admin reasons and lower when needed.

Reply:
done